examples/diesel-sqlite/main.rs

```diff
@@ -6,12 +6,60 @@ use diesel::{
 use jiff::civil;
 use jiff_diesel::ToDiesel;
 
+mod schema {
+    diesel::table! {
+        datetimes {
+            id -> Integer, // Diesel tables require an ID column.
+            ts -> TimestamptzSqlite,
+            dt -> Timestamp,
+            d -> Date,
+            t -> Time,
+        }
+    }
+
+    diesel::table! {
+        nullable_datetimes {
+            id -> Integer, // Diesel tables require an ID column.
+            ts -> Nullable<TimestamptzSqlite>,
+            dt -> Nullable<Timestamp>,
+            d -> Nullable<Date>,
+            t -> Nullable<Time>,
+        }
+    }
+}
+
 fn main() -> anyhow::Result<()> {
     let tmpfile = tempfile::NamedTempFile::new()?;
     let path = tmpfile.path().to_str().context("invalid temporary file")?;
     let mut conn = SqliteConnection::establish(path)?;
```

Comment:
```suggestion
    let tmpfile = tempfile::NamedTempFile::new()?;
    let path = tmpfile.path().to_str().context("invalid temporary file")?;
    let mut conn = SqliteConnection::establish(path)?;
```

I would like just use a in memory database here instead. So just `SqliteConnection::establish(":memory:")?;`